### DISCUSSION TITLE
Apparition Bandeau accepter/refuser

### DISCUSSION THREAD

**Discussion by linuxmrcmd** [AmauriC/tarteaucitron.js#1129] [Ideas]
Ce bandeau apparait pour tous les scripts qui sont à la racine du site.
Si un internaute consulte directement par un script autorisé amis présent dans un dossier, ce bandeau n'apparait pas
Je précise que aucune acceptation ou aucun refus n'a pas été confirmé
En résumé, c'est d'avoir l'apparition de ce bandeau dans d'autres parties du site utilisées directement

**Comment by AmauriC** [comment]
Je ne suis pas sûr de comprendre, possible d'avoir un exemple avec des urls?

**Reply by AmauriC** [reply]
Sur la page fiches.php c'est le chemin d'accès au script tarteaucitron qui est incorrect:
```
<script src="inc/tarteaucitron/tarteaucitron.js"></script>
```

Il manque un slash au début:

```
<script src="/inc/tarteaucitron/tarteaucitron.js"></script>
```

**Reply by linuxmrcmd** [reply]
ok, merci
